src/index.ts

```diff
@@ -135,6 +136,9 @@ export class GoogleToken {
       }
       case 'application/x-pkcs12': {
         // *.p12 file
+        if (!getPem) {
+          getPem = (await import('google-p12-pem')).getPem;
```

Comment:
Is dynamic import already supported? I know TS supports it but I thought it worked only with `esnext`. How does this transpile to JS? Just `require('...')`?

Reply:
This change would need a common with the rationale for why deferred import is necessary.

Reply:
I'm as surprised as you  are that this worked in node.js 4 🤷‍♂️   It looks like it does melt down to a require:

```js
if (!!getPem) return [3 /*break*/, 7];
  return [4 /*yield*/, Promise.resolve().then(function () { return require('google-p12-pem'); })];
```

@ofrobots good call, added a comment.